crypto/keyring/info.go

```diff
@@ -176,35 +176,21 @@ func (i offlineInfo) GetPath() (*hd.BIP44Params, error) {
 	return nil, fmt.Errorf("BIP44 Paths are not available for this type")
 }
 
-type multisigPubKeyInfo struct {
-	PubKey cryptotypes.PubKey `json:"pubkey"`
-	Weight uint               `json:"weight"`
-}
-
 // multiInfo is the public information about a multisig key
 type multiInfo struct {
-	Name      string               `json:"name"`
-	PubKey    cryptotypes.PubKey   `json:"pubkey"`
-	Threshold uint                 `json:"threshold"`
-	PubKeys   []multisigPubKeyInfo `json:"pubkeys"`
+	Name   string             `json:"name"`
+	PubKey cryptotypes.PubKey `json:"pubkey"`
```

Comment:
I believe this struct is amino-marshalled and stored in the keyring backend, so this breaks backward-comptability of the keyring.

Reply:
It's used both in CLI and keyring storage. We are removing a fileds here, not changing it. It won't break unmarshalling.

Reply:
Are you sure about this?

If a user already has a keyring backend where the old multiInfo `bz` were stored, then, unmarshalling those `bz` into this new `multiInfo` will error.

Reply:
I will test it.

Reply:
Hmm, I thought I responded on this comment.
So... yes, you are right. `legacy.Cdc.UnmarshalBinaryLengthPrefixed` fails to decoded a subset of an encoded object. Thanks for catching. I added back the `multisigPubKeyInfo` and marked it deprecated. 
Also created an issue: https://github.com/cosmos/cosmos-sdk/issues/8323 to track the keyring migration.